hazelcast/src/main/java/com/hazelcast/internal/metrics/metricsets/RuntimeMetricSet.java

```diff
@@ -44,11 +50,12 @@ public static void register(MetricsRegistry metricsRegistry) {
         Runtime runtime = Runtime.getRuntime();
         RuntimeMXBean mxBean = ManagementFactory.getRuntimeMXBean();
 
-        metricsRegistry.registerStaticProbe(runtime, "runtime.freeMemory", MANDATORY, Runtime::freeMemory);
-        metricsRegistry.registerStaticProbe(runtime, "runtime.totalMemory", MANDATORY, Runtime::totalMemory);
-        metricsRegistry.registerStaticProbe(runtime, "runtime.maxMemory", MANDATORY, Runtime::maxMemory);
-        metricsRegistry.registerStaticProbe(runtime, "runtime.usedMemory", MANDATORY, JVMUtil::usedMemory);
-        metricsRegistry.registerStaticProbe(runtime, "runtime.availableProcessors", MANDATORY, Runtime::availableProcessors);
-        metricsRegistry.registerStaticProbe(mxBean, "runtime.uptime", MANDATORY, RuntimeMXBean::getUptime);
+        metricsRegistry.registerStaticProbe(runtime, RUNTIME_FULL_METRIC_FREE_MEMORY, MANDATORY, Runtime::freeMemory);
```

Comment:
Is it possible to specify `BYTES` unit here and for other memory related probes in this class?

Reply:
And same :)